src/unfold/compat.py

```diff
@@ -0,0 +1,15 @@
+import django
+from django.contrib.admin.views.main import ERROR_FLAG, PAGE_VAR
+from django.contrib.admin.views.main import ChangeList as DjangoChangeList
+
+if django.VERSION < (5, 0):
```

Comment:
Do we really need to check for Django version? Isn't it better just to check if there is `filter_params`attribute?

Reply:
> Do we really need to check for Django version? Isn't it better just to check if there is `filter_params`attribute?

Checking for the Django version explicitly is a great helper when getting rid of outdated code in the future - as soon as Django 4.2 is deprecated.

Otherwise it will happen that the helper ChangeList will stay there way too long ... just my two cents.